server/controllers/BasicServices.js

```diff
@@ -86,25 +87,26 @@ module.exports.startApplicationInGenericRepresentation = async function startApp
   ExecutionAndTraceService.recordServiceRequest(xCorrelator, traceIndicator, user, originator, req.url, responseCode, req.body, responseBodyToDocument);
 };
 
-module.exports.informAboutPrecedingRelease = function informAboutPrecedingRelease (req, res, next, user, originator, xCorrelator, traceIndicator, customerJourney) {
+module.exports.informAboutPrecedingRelease = async function informAboutPrecedingRelease (req, res, next, user, originator, xCorrelator, traceIndicator, customerJourney) {
   let startTime = process.hrtime();
   let responseCode = responseCodeEnum.code.OK;
   let responseBodyToDocument = {};
-  BasicServices.informAboutPrecedingRelease(user, originator, xCorrelator, traceIndicator, customerJourney)
-  .then(async function (responseBody) {
-    responseBodyToDocument = responseBody;
-    let responseHeader = await RestResponseHeader.createResponseHeader(xCorrelator, startTime, req.url);
-    RestResponseBuilder.buildResponse(res, responseCode, responseBody, responseHeader);
-  })
-  .catch(async function (responseBody) {
-    let responseHeader = await RestResponseHeader.createResponseHeader(xCorrelator, startTime, req.url);
-    let sentResp = RestResponseBuilder.buildResponse(res, undefined, responseBody, responseHeader);
-    responseCode = sentResp.code;
-    responseBodyToDocument = sentResp.body;
-  });
-ExecutionAndTraceService.recordServiceRequest(xCorrelator, traceIndicator, user, originator, req.url, responseCode, req.body, responseBodyToDocument);
+  await BasicServices.informAboutPrecedingRelease(OLD_RELEASE_FORWARDING_NAME)
```

Comment:
could you please test disposeRemaindersOfDeregisteredApplication , inquireBasicAuthRequestApprovals and UpdateClientOfSubsequentRelease

Reply:
yes tested working as expected